404.php

```diff
@@ -47,10 +47,10 @@
 					<?php
 					/* translators: %1$s: smiley */
 					$archive_content = '<p>' . sprintf( __( 'Try looking in the monthly archives. %1$s', 'lean' ), convert_smilies( ':)' ) ) . '</p>';
-					the_widget( 'WP_Widget_Archives', 'dropdown=1', "after_title=</h2>$archive_content" );
```

Comment:
This one seems interesting. I didn't change this and almost didn't touch this file. Even this line is still there in my local environment. @mitogh